src/applications/hca/components/FormPages/DependentInformation.jsx

```diff
@@ -268,6 +272,7 @@ const DependentInformation = props => {
               text={`Cancel ${action.label} this dependent`}
               onClick={handlers.showConfirm}
               secondary
+              id="cancel-adding-dependent"
```

Comment:
Can we maybe change this ID name to something like `hca-modal-cancel` or something along those lines? Make it just a tick more generic and prefix with `hca-` to avoid ever having a naming conflict.

Reply:
yes, will do!